File: WalekV01.Providers.Sql/Models/Video.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace WalekV01.Providers.Sql.Models
{
    [Table("Video")]
    public class Video
    {
        [Key]
        public int Id { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public int Duration { get; set; }
        public int? Episode { get; set; }
        public string Producer { get; set; }
        public DateTime ReleaseDate { get; set; }
        public string? ImageUrl { get; set; }
        public int UserId { get; set; }
        public int GenderId { get; set; }
        public IEnumerable<VideoCategories> Categories { get; set; }

    }
}

File: WalekV01.Presentation.API/ClientApp/src/app/models/video.ts
import { Categories } from './categories';
import { VideoCategories } from './video-categories';
export interface Video {
    id?: number;
    title: string;
    description: string;
    duration: number;
    episode? : number;
    producer : string;
    releaseDate : Date;
    imageUrl: string;
    categories: VideoCategories[];
    genderId? : number;
}
